lib/pymedphys/docs/lib/howto/get-started.rst

```diff
@@ -6,21 +6,61 @@ Install Python
 ==============
 
 In order to make use of the PyMedPhys library, you'll need Python installed on
-your workstation. On Windows we recommend that you install the
-Anaconda Python distribution. Download the latest Anaconda
-version from `here <https://www.anaconda.com/products/individual#Downloads>`__.
+your workstation. Below are some recommended instructions for installing Python
+based upon your OS.
 
-.. note::
+Windows 10 or 11
+----------------
 
-    When installing Anaconda make sure to install it for your user only, and
-    tick the option "add to path".
+Open up a command prompt and type:
 
-.. image:: ../../img/add_anaconda_to_path.png
+.. code:: bash
+
+    python
+
+If python isn't already installed this will open the Windows store. At this
+point you can click the "Get" button. It will ask you to sign in, but you can
+skip the sign in step. It will still install.
+
+Once installed, open a new command prompt and type:
+
+.. code:: bash
+
+    python
```

Comment:
No biggie, but could instead suggest a command that doesn't require a subsequent exit? Something like `python --version`?

Reply:
Implemented :slightly_smiling_face: